File: cpp/quesn15.py
# Define a function that can receive two integer numbers in string form and compute their sum and then print it in console.
def sum():
	x,y=(input("Enter two integers\n")).split()
	n1=int(x)
	n2=int(y)
	s=n1+n2
	print(s)

sum()

File: c/quesn15.c
#include <iostream>
#include <string>
using namespace std;

string adderFunc(string s1, string s2)
{
    double n1 = std::stod(s1);
    double n2 = std::stod(s2);
    double sum = n1 + n2;

    return std::to_string(sum);
}
int main()
{
    string s1, s2;
    cout << "Please enter string 1 : ";
    cin >> s1;
    cout << "Please enter string 2 : ";
    cin >> s2;

    cout << "The sum of these two numbers is : " << adderFunc(s1, s2) << endl;
    return 0;
}
